src/modules/eventListeners.js

```diff
@@ -0,0 +1,63 @@
+import popup from "./popup.js";
+import getMovies from "./getMovies.js";
+import moviesCounter from "./counters/moviesCounter.js";
+
+const pop = document.querySelector(".pop");
+const body = document.querySelector("body");
+
+const movieSection = document.querySelector(".main");
+
+document.addEventListener("DOMContentLoaded", async () => {
+  //   addLikes();
```

Comment:
Please delete the comment-like code from your work as it is good coding practice.